src/main/java/fi/fmi/avi/model/swx/VerticalLimitsImpl.java

```diff
@@ -0,0 +1,58 @@
+package fi.fmi.avi.model.swx;
+
+import org.inferred.freebuilder.FreeBuilder;
+
+import java.io.Serializable;
+import java.util.Objects;
+import java.util.Optional;
+
+@FreeBuilder
+public abstract class VerticalLimitsImpl implements VerticalLimits, Serializable {
+
+    private static final long serialVersionUID = 456150576462009061L;
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static VerticalLimits none() {
+        return builder().build();
+    }
```

Comment:
I think this method should return a constant. There's no need to build a new immutable instance on every invocation.